RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate readMessages to TS

## Proposed changes

Migrate `readMessages` to TS
## Issue(s)	


## How to test or reproduce

With two accounts:
1. Send a message to `account 2`
2. Open the chat with `account 2` and then leave.
3. Restart the app.

It should be marked as read.
## Screenshots

## Types of changes




## Checklist



## Further comments
